server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -811,4 +829,24 @@ public int getLivePort() {
   public ServiceLock getLock() {
     return monitorLock;
   }
+
+  public InformationFetcher getInformationFetcher() {
+    return fetcher;
+  }
+
+  @Override
+  public void onOpened(Connection connection) {
+    log.info("New connection event");
```

Comment:
Probably should not log new connection events at info level. I'm guessing this was added during testing of this connection listener, but is no longer needed.

```suggestion
```